# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kasc/openal/alsa_header.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./linux/src/arch/alsa/alsa.c.orig	2003-04-30 00:03:20.000000000 +0100
+++ ./linux/src/arch/alsa/alsa.c	2003-04-30 00:03:40.000000000 +0100
@@ -21,6 +21,7 @@
 #include "arch/interface/interface_sound.h"
 #include "arch/alsa/alsa.h"
 
+#include <sound/asound.h>
 #include <alsa/asoundlib.h>
 
 #include "al_config.h"
